src/js/components/CharityPage.jsx

```diff
@@ -56,24 +56,41 @@ const CharityPage = () => {
 	const impactColumn = (
 		<Col md="7" xs="12" className="column impact-column">
 			<div className="header">
+				{charity.logo? <div><img src={charity.logo} alt='Charity logo' className='charity-logo'/></div> : null}
 				<h1 className="charity-name">
 					{charity.displayName || charity.name} <small><EditLink charity={charity} /></small>
 				</h1>
-				<CharityTags className="why-tags" tagsString={charity.whyTags} />
-				<CharityTags className="where-tags" tagsString={charity.whereTags} />
+				<div className='description-short'>
+					{charity.summaryDescription? <MDText source={charity.summaryDescription} /> : null}
+				</div>
+				<CharityTags whyTagsString={charity.whyTags} whereTagsString={charity.whereTags} />
 			</div>
 			<CharityDonate charity={charity} />
+			<div className="header">
+				<h2 className="charity-section-title">
+					About {charity.displayName || charity.name}
+				</h2>
+			</div>
+			<div className='charity-section-text'>
+				<div className='description-short'>
```

Comment:
Sadly, we should have some defensive code here for the no-description case. E.g. http://local.sogive.org/#charity?charityId=food-train

Reply:
I do check if summaryDescription exists & only insert it if so on line 76, and it looks ok.

However I'm not checking if tags exist and that looks weird. (see attached). Will fix that instead.
![Screenshot from 2020-09-20 18-28-24](https://user-images.githubusercontent.com/1918555/93717652-1d50d600-fb6f-11ea-827c-e0397858b00e.png)

Reply:
(Done)